vad_transcriber/audio/2830-3980-0043.txt

```diff
@@ -0,0 +1 @@
+experience proves this 
```

Comment:
Why do you add all this `audio/` subdirectory, including this text file, licenses + attribution and audio samples? There's no need for them?